.gitlab-ci.yml

```
@@ -165,7 +165,7 @@ before_script:
   needs:
     - not-a-real-job
   script:
-    - SPHINXENV='COQBIN="'"$PWD"'/_install_ci/bin/"'
+    - SPHINXENV='OCAMLPATH="'"$PWD"'/_install_ci/lib/:$(OCAMLPATH)" COQBIN="'"$PWD"'/_install_ci/bin/"'
```

This is expanding OCAMLPATH by make which seems kinda weird.
I went overboard with the quoting when I made this job, 
```suggestion
    - SPHINXENV="OCAMLPATH='$PWD/_install_ci/lib/:$OCAMLPATH' COQBIN='$PWD/_install_ci/bin/'"
```
should work fine, or even without the single quotes since I don't think there are special characters on the CI machine's paths.